camkes/ast/liftedast.py

```diff
@@ -57,14 +59,15 @@ def assembly(self):
 
     def filter(self, function=None):
         if function is None:
-            function = lambda x: x is not None
+            def function(x): return x is not None
```

Comment:
This is more then just a style change, please make this also a separate commit.

Reply:
Done.